python/metric_pipeline_tasks/MatchedCatalogMeasureTasks.py

```diff
@@ -176,6 +181,73 @@ def run(self, matchedCatalog, metric_name):
         return Struct(measurement=Measurement(metric_name, np.median(rmsDistances.to(u.marcsec))))
 
 
+class ADxTask(Task):
+    ConfigClass = AMxTaskConfig
+    _DefaultName = "ADxTask"
+
+    def __init__(self, config: AMxTaskConfig, *args, **kwargs):
+        super().__init__(*args, config=config, **kwargs)
```

Comment:
Why not just use the `config` directly in the run method as is done in `AMxTask`?  That would decrease the duplication between `ADxTask` and `AFxTask`.